create_infotexts.py

```diff
@@ -387,7 +368,23 @@ def process_depicted_place(self, place_string, places_mapping, desc_string):
                 self.content_cats.append(places_mapping[place_string]["commonscat"])
 
         else:
-            place_as_wikitext += place_string
+            place_matches = []
+            for place in places_mapping:
+                if place.lower() in desc_string.lower():
+                    if places_mapping[place]["wikidata"]:
```

Comment:
I looked into it at and indeed it was due to errors.

Now it leaves only one erroneous double-hit which I'll handle manually per https://phabricator.wikimedia.org/T163669